inference-engine/thirdparty/clDNN/src/graph_optimizer/remove_redundant_reorders.cpp

```diff
@@ -363,6 +363,59 @@ void remove_redundant_reorders::run(program_impl& p) {
         p.remove_if_dangling(*node);
     }
 
+    // This pass removes reorder for Convolution b_fs_yx_fsv16 -> bfyx
+    if (enable_reorder_fusing) {
+        itr = p.get_processing_order().begin();
+        while (itr != p.get_processing_order().end()) {
+            auto& node_ptr = *itr++;
+            if (!node_ptr->is_type<reorder>() ||
+                !node_ptr->is_in_data_flow() ||
+                node_ptr->get_dependencies().size() != 1) {
```

Comment:
I think you also need to check that reorder doesn't have fused ops / fused activations

Reply:
I don't get which checking is related with "fused ops/fused activations".

Reply:
I guess vladimir mentioned something like this : !node_ptr->get_fused_activations_funcs().empty())

Reply:
fixed